pkg/agent/controller/types.go

```diff
@@ -39,7 +39,7 @@ const (
 	portConflictReason = "ConflictingPorts"
 )
 
-var brokerResyncePeriod = time.Minute * 2
+var BrokerResyncePeriod = time.Minute * 2
```

Comment:
While you’re at it, you could fix the typo ;-)

```suggestion
var BrokerResyncPeriod = time.Minute * 2
```